CHANGELOG.md

```diff
@@ -3,6 +3,60 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## v5.12.1
+
+_Jun 9, 2022_
+
+We'd like to offer a big thanks to the 10 contributors who made this release possible. Here are some highlights ✨:
+
+TODO INSERT HIGHLIGHTS
```

Comment:
As far as I see most of it is bug fixes. Is there something specific we would like to add here?

Reply:
For the pickers, the main new thing is the warning on the lab (not directly related to this release though).

Or #5060 (@alexfauquette)